core/templates/dev/head/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts

```diff
@@ -825,6 +833,13 @@ oppia.directive('conversationSkin', [
                 return;
               }
             }
+            if (ENABLE_SOLICIT_ANSWER_DETAILS_FEATURE) {
+              if (LearnerAnswerInfoService.initLearnerForAnswerInfo(
```

Comment:
I think maybe just `init` is fine, or `initLearnerAnswerInfo`

Reply:
Done